src/current/_includes/v23.1/cdc/work-distribution-setting.md

```diff
@@ -0,0 +1 @@
+{% include_cached new-in.html version="v23.1.23" %} You can enable the `changefeed.random_replica_selection.enabled` cluster setting to change the way in which a changefeed distributes work across the cluster. For the cluster setting to take effect on changefeed jobs, [pause]({% link {{ page.version.version }}/pause-job.md %}) the changefeed, enable `changefeed.random_replica_selection.enabled`, and then [resume]({% link {{ page.version.version }}/resume-job.md %}) the changefeed. With `changefeed.random_replica_selection.enabled` set to `true`, the job will evenly distribute changefeed work across the cluster by assigning it to any [replica]({% link {{ page.version.version }}/architecture/replication-layer.md %}) for a particular range.
```

Comment:
A point of clarification. The operator doesn't need to pause, set the setting, and then resume. They can set the setting and then pause and resume (though they could do it the other way as well). I think it's better to describe the latter sequence, because the former is mroe onerous if the operator has many changefeeds.

Reply:
OK, sounds good. Changed the v23.1 and v23.2 include files. Switched the ordering of sentences around too.